data/schemas/edges/variants_proteins.pQTL.json

```diff
@@ -1,95 +1,84 @@
-        },
-        "source": {
-            "type": "string",
-            "description": "The source of the variant to protein edge",
-            "enum": ["UKB"]
-        },
-        "source_url": {
-            "type": "string",
-            "description": "The source url of the variant to protein edge",
-            "enum": ["https://metabolomips.org/ukbbpgwas/"]
-        },
-        "name": {
-            "type": "string",
-            "description": "The name of the variant to protein edge",
-            "enum": ["associated with levels of"]
-        },
-        "inverse_name": {
-            "type": "string",
-            "description": "The inverse name of the variant to protein edge",
-            "enum": ["level associated with"]
-        },
-        "method": {
-            "type": "string",
-            "description": "The method of the variant to protein edge",
-            "enum": ["ontology_terms/BAO_0080027"]
+    "allOf": [
+        {
+            "$ref": "edges.base.json"
+        },
+        {
+            "type": "object",
+            "properties": {
+            "_key": {
+                "type": "string",
+                "description": "Unique identifier for the variant to protein edge",
+                "example": "NC_000017.11:7166092:G:A_ENSP00000491556_UKB"
+            },
+            "_from": {
+                "example": "variants/NC_000001.11:10000:T:A",
+                "pattern": "^variants/",
+                "collections": ["variants"]
+            },
+            "_to": {
+                "example": "proteins/ENSP00000456328",
+                "pattern": "^proteins/",
+                "collections": ["proteins"]
+            },
+            "rsid": {
+                "type": ["string", "null"],
+                "description": "The rsid of the variant to protein edge",
+                "example": "rs1260326"
+            },
+            "label": {
+                "enum": ["pQTL"]
+            },
+            "log10pvalue": {
+                "type": "number",
+                "description": "The log10pvalue of the variant to protein edge"
+            },
+            "beta": {
+                "type": "number",
+                "description": "The beta of the variant to protein edge"
+            },
+            "se": {
+                "type": "number",
+                "description": "The se of the variant to protein edge"
+            },
+            "class": {
+                "enum": ["cis", "trans"]
+            },
+            "gene": {
+                "type": ["string", "null"],
+                "description": "The gene of the variant to protein edge",
+                "example": "genes/ENSG00000165841"
+            },
+            "gene_consequence": {
+                "type": ["string", "null"],
+                "description": "The gene consequence of the variant to protein edge",
+                "example": "intergenic_variant"
+            },
+            "biological_context": {
```

Comment:
we have a relevant ticket on this field here https://igvf.atlassian.net/browse/DSERV-871, to fix this field to be ontology_terms/XXX. but we can do this later.